batch/batch/aioclient.py

```diff
@@ -105,11 +106,21 @@ def __init__(self, client, id, attributes):
 
 
 class BatchClient:
-    def __init__(self, session, url=None):
+    def __init__(self, session, url=None, token_file=None, token=None, headers=None):
         if not url:
             url = 'http://batch.default'
         self.url = url
         self._session = session
+        if token is None:
+            token_file = (token_file or
```

Comment:
I think one of the paths should be the path that notebook mounts, so that no additional secret or env variable needs to be set for notebook workers to use batch. 

<img width="542" alt="Screenshot 2019-04-10 16 49 22" src="https://user-images.githubusercontent.com/5543229/55913044-af4e6400-5bb1-11e9-820d-2867d50da6e3.png">